packages/project/src/utils/convertFromV3ToV4.ts

```diff
@@ -196,6 +196,7 @@ const convertV3CropToV4Crop = (
 export const convertV3FieldToV4Field = (v3Field: FieldV3): FieldV4 => {
   return {
     ...v3Field,
+    externalId: crypto.randomUUID(),
```

Comment:
Since the v3 spec does not have a reliable unique identifier, I took the approach of generating one during v3 --> v4 conversion. 

This approach is not perfect, since it means converting a file twice will yield outputs with different IDs, but the other approaches for creating a derivable unique ID all made assumptions around what constitutes a "unique" field. 

I think this approach is the best, so that we get a reliable v4 output without making assumptions about the source v3 file